packages/@uppy/provider-views/src/index.js

```diff
@@ -358,14 +358,14 @@ module.exports = class ProviderView {
       state = this.plugin.getPluginState()
       state.selectedFolders[folderId] = { loading: false, files: files }
       this.plugin.setPluginState({ selectedFolders: folders })
-      const dashboard = this.plugin.uppy.getPlugin('Dashboard')
```

Comment:
I think we have to keep this as a fallback for now, else it's a subtly breaking change :/

Reply:
Hm 🤔 But that would only break if you are upgrade to the latest Dashboard, but not Core, right? 

Cause this is also kind of a bug, the Dashboard might have a different id, and it does with React, for example.

Reply:
I'm thinking of the case where a user manually configured this string in the Dashboard plugin.

It is really tiny, maybe we can just do it in a minor and mention it in a blog post :woman_shrugging:

Reply:
Yeah, let’s do that, I’ve started adding a condition and really feel like it might be more trouble than it’s worth.